src/containers/layout/GuestLayout.js

```diff
@@ -15,9 +16,9 @@ const GuestLayout = () => {
       <Route element={ <Example /> } name="home" path={ routes.about.route } />
       <Route element={ <AuthPolicy /> } path={ errors.authPolicy.route } />
       <Route element={ <BadRequest /> } path={ errors.badRequest.route } />
+      <Route element={ <CookiePolicy /> } name="FindMentor" path={ routes.privacyPolicy.route } />
       <Route element={ <NotFound /> } path={ errors.notFound.route } />
       <Route element={ <InternalServerError /> } path={ errors.internalServerError.route } />
-      <Route element={ <Navigate to={ errors.notFound.route }  /> } path='*' />
```

Comment:
return back this route

Reply:
![image](https://user-images.githubusercontent.com/57355852/182161035-080f487d-a279-4d61-911d-faa22c1be061.png)
fix code smells

Reply:
fix